apps/web/.env.template

```
@@ -10,6 +10,8 @@ NEXT_PUBLIC_NWS_SERVER=wss://enotify.ecency.com
 NEXT_PUBLIC_GMAPS_API_KEY=
 NEXT_PUBLIC_GMAPS_MAP_ID=
 NEXT_PUBLIC_THREESPEAK_EMBED_ENDPOINT=https://embed.3speak.tv
+# Stripe publishable key (public). When empty, card payment for Points is hidden.
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY key should go before the NEXT_PUBLIC_THREESPEAK_EMBED_ENDPOINT key

(UnorderedKey)